chainer/functions/pooling/upsampling_2d.py

```diff
@@ -203,7 +203,7 @@ def upsampling_2d(
 
         >>> p = F.MaxPooling2D(2, 2)
         >>> with chainer.using_config('use_cudnn', 'never'):
-        ...     pooled_x = p(x)
+        ...     pooled_x = p.apply((x,))[0]
```

Comment:
Why it uses `FunctionNode` object directly?

Reply:
It wants to use ` p.indexes` in above line.

Reply:
How about `pooled_x.creator.indexes`?

Reply:
OK, I will fix.